modules/javafx.controls/src/test/java/test/javafx/scene/control/TreeTableViewTest.java

```diff
@@ -6987,4 +6991,93 @@ public void testRemoveTreeItemFromCollapsedAncestorKeepsSelectedItem() {
         assertEquals(0, table.getFocusModel().getFocusedIndex());
         assertEquals("Node 0", table.getFocusModel().getFocusedItem().getValue());
     }
+
+    // See JDK-8087673
+    @Test
```

Comment:
would it be possible to add a short description of what the tests is trying to verify?